src/librados/librados_c.cc

```diff
@@ -2545,6 +2545,7 @@ static void rados_aio_getxattr_complete(rados_completion_t c, void *arg) {
   }
   cdata->user_completion.finish(rc);
   delete cdata;
+  ((librados::AioCompletionImpl*)c)->put();
```

Comment:
nit, can we follow the same patter of 

https://github.com/ceph/ceph/blob/c28876f0084d583de0550b105c2ae45222fd30e8/src/librados/IoCtxImpl.cc#L1178-L1179

i.e. call `put()` before `delete cdata`? it's not big deal, but just for the sake of consistency.

Reply:
@tchaikov Synced.